src/Microsoft.IdentityModel.Tokens.Saml/Saml/LogMessages.cs

```diff
@@ -51,6 +51,46 @@ internal static class LogMessages
         internal const string IDX11110 = "IDX11110: The SamlSecurityToken must have a value for its Assertion property.";
 
+        internal const string IDX11111 = "IDX11111: When reading '{0}', '{1}' was not a Absolute Uri, was: '{2}'.";
+        internal const string IDX11112 = "IDX11112: Exception thrown while reading '{0}' for SamlSecurityToken. Inner exception: '{1}'.";
+        internal const string IDX11114 = "IDX11114: Unable to read SamlSecurityToken. Expecting XmlReader to be at element: '{0}', found 'Empty Element'";
+        internal const string IDX11115 = "IDX11115: Unable to read for SamlSecurityToken. Element: '{0}' as missing Attribute: '{1}'.";
+        internal const string IDX11116 = "IDX11116: Unable to read for SamlSecurityToken. MajorVersion must be '1' was: '{0}'.";
+        internal const string IDX11117 = "IDX11117: Unable to read for SamlSecurityToken. MinorVersion must be '1' was: '{0}'.";
+        internal const string IDX11118 = "IDX11118: Unable to read condition : '{0}'. SamlSecurityToken only support AudienceRestrictionCondition and DoNotCacheCondition.";
+        internal const string IDX11119 = "IDX11119: Unable to read for SamlSecurityToken. Element: '{0}' as missing element: '{1}' when reading element: '{2}'.";
+        internal const string IDX11120 = "IDX11120: Unable to read SamlSecurityToken. Expecting XmlReader to be at element: '{0}', found: '{1}'.";
+        internal const string IDX11121 = "IDX11121: Invalid SAML AssertionID: '{0}'.";
+        internal const string IDX11122 = "IDX11122: Exception thrown while reading '{0}' for Saml2SecurityToken. Inner exception: '{1}'.";
+        internal const string IDX11123 = "IDX11123: Unable to read SamlSecurityToken. Expecting XmlReader to be at element: '{0}', found 'Empty Element'";
+        internal const string IDX11124 = "IDX11124: Unable to read SamlSecurityToken. Unexpecting element: '{0}' in element: '{1}' has been found";
+        internal const string IDX11125 = "IDX11125: Unable to read SamlSecurityToken. Missing data in element: '{0}'";
+        internal const string IDX11126 = "IDX11126: A <saml:'{0}'> contained unrecognized content: '{1}'.The schema allows arbitrary XML elements on this element without explicit schema extension.To handle the content, extend SamlSerializer.";
+        internal const string IDX11127 = "IDX11127: Saml unable to load Conditions";
+        internal const string IDX11128 = "IDX11128: Saml unable to load Advice";
+        internal const string IDX11129 = "IDX11129: Saml unable to load Statement";
+        internal const string IDX11130 = "IDX11130: When reading '{0}', no Statements were found.";
+        internal const string IDX11131 = "IDX11131: Unable to read for SamlSecurityToken. the AttributeStatement does not contain any Attributes.";
+        internal const string IDX11132 = "IDX11132: Unable to read for SamlSecurityToken. the Attribute does not contain any AttributeValue.";
```

Comment:
grammer: to read for SamlSecurityToken. the -> to read the SamlSecurityToken, the